integration-tests/container/Makefile

```diff
@@ -0,0 +1,20 @@
+BASE_PATH = .
+include Makefile-constants.mk
+
+SHELL := /bin/bash
+
+.PHONY: build-all
+build-all:
+	@set -eou pipefail; \
+	mapfile -t container_dirs < <(ls -d */); \
```

Comment:
I think mapfile is linux specific, so this may break for those of us who are using MacOS. I think this can also be achieved in pure (gnu) make

```
DIRS := $(sort $(dirs $(wildcard ./*))
```

And possibly the following, though I've not tested it

```
TARGETS := build build-and-push
DIRS := $(sort $(dirs $(wildcard ./*))

$(TARGETS): $(DIRS)
$(DIRS):
    $MAKE -C $@ $(MAKECMDGOALS)

.PHONY: $(TARGETS) $(DIRS)
```

Reply:
Your suggestion didn't work, but I changed it a bit and now it does.